docs/javascript/v2/how--to-guides/extend-capabilities/plugins/virtual-background.mdx

```diff
@@ -309,3 +309,18 @@ function VirtualBackground() {
 ```
 
 </Tab>
+
+
+## Recommendations
+
+- Use background images that match the video publish aspect ratio set from dashboard template.
+  Eg: For 4:3 aspect ratio, image should be a minimum of 640x480 or higher.
+- Image may look stretched depending on video element size, so it should be closer to the video element size
```

Comment:
not sure i understand this. should i use the video element dimensions (which can be different for different clients), or the capture settings of my template?

Reply:
video element atleast in our case is in some aspect ratio. video tile ideally should be of same aspect ratio as publish. will update this.